.circleci/config.yml

```diff
@@ -560,12 +625,24 @@ workflows:
       - publish_release:
           context: "OC Common"
           requires:
-            - go_lint
             - go_test
             - go_test_fuse
+            - go_test_metrics
             - build_images
           filters:
             tags:
               only: /^v.*/
             branches:
               ignore: /.*/
+
+  weekly:
+    triggers:
+      - schedule:
+          cron: "5 4 * * 6"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - push_base:
```

Comment:
Might be better to build the base image only on merge into master instead of a cron job.

Reply:
Done as discussed